src/test/java/org/minions/devfund/yuri/linkedlist/RemoveMaximumValuesTest.java

```diff
@@ -0,0 +1,206 @@
+package org.minions.devfund.yuri.linkedlist;
+
+import java.util.LinkedList;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * // Source code recreated from a .class file by IntelliJ IDEA.
+ * // (powered by Fernflower decompiler)
+ */
+public class RemoveMaximumValuesTest {
+
+    private static final int LISTSIZE1 = 3;
+    private static final int LISTSIZE2 = 4;
+
+    /**
+     * test.
+     */
+    @Test
+    public void testNull() {
+        try {
+            LinkedListUtils.removeMaximumValues((LinkedList) null, 2);
+        } catch (Exception var2) {
+            Assert.fail("removeMaximumValues throws " + var2 + " when input LinkedList is null");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "removeMaximumValues throws " 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

Reply:
done